addons/test_mail_full/tests/test_rating.py

```diff
@@ -5,12 +5,12 @@
 
 from odoo.addons.test_mail_full.tests.common import TestMailFullCommon, TestMailFullRecipients
 from odoo.tests import tagged
-from odoo.tests.common import users, warmup
+from odoo.tests.common import HttpCase, users, warmup
 from odoo.tools import mute_logger
 
 
 @tagged('rating')
-class TestRatingFlow(TestMailFullCommon, TestMailFullRecipients):
+class TestRatingFlow(HttpCase, TestMailFullCommon, TestMailFullRecipients):
```

Comment:
Would make a dedicated class for httpcase (that can inherit from TestRatingFlow), to avoid adding too much stuff on standard tests more low level

Reply:
Done :+1: